src/webserver.cc

```diff
@@ -28,8 +27,13 @@ RequestHandler* Webserver::matchRequestWithHandler(const Request& req) {
 
 std::string Webserver::processRawRequest(std::string& reqStr) {
     auto req = Request::Parse(reqStr);
+    if (req == nullptr) {
+        return "";
+    }
     Response resp;
 
+    std::cout << "request to '" << req->uri() << "'\n";
```

Comment:
Yeah it's easier to see what requests the server is handling and not handling, and if it gets stuck somewhere weird.